games-emulation/retroarch/retroarch-1.7.5.recipe

```diff
@@ -8,10 +8,9 @@ COPYRIGHT="2010-2018 The RetroArch Team"
 LICENSE="GNU GPL v3"
 REVISION="1"
 SOURCE_URI="https://github.com/libretro/RetroArch/archive/v$portVersion.tar.gz"
-CHECKSUM_SHA256="1c095076e2a9d2ec4e501176b3963ea2a831c1e5f025bb25386491272f14de40"
+CHECKSUM_SHA256="89dae3646e4979b8685cae987eaa09816ac94bfb10d2636a2e7a8eede5fcbfa3"
 SOURCE_FILENAME="retroarch-$portVersion.tar.gz"
 SOURCE_DIR="RetroArch-$portVersion"
-PATCHES="retroarch-$portVersion.patchset"
 
 ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
```

Comment:
Thanks @kwyxz for your PR!

Could you please apply this tiny change?
```diff
 ARCHITECTURES="!x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
 SECONDARY_ARCHITECTURES="x86"
+fi
```
This will be useful on x86_64 hybrid. It means that on that arch there is no `retroarch_x86`. We do this on recipes which do not provide any lib. `retroarch_x86` will only be available on x86_gcc2 hybrid. On x86 and x86_64 we don't need `retroarch_x86` because we have a working `retroarch`.
Thanks!

(BTW, I'll build the recipe asap.)

Reply:
I've just built the recipe on x86_gcc2 (using x86 secondary arch) and x86_64 and everything looks OK.

I'll push the tiny change described above and then merge !

Reply:
Hmm, I just noticed we had a tiny mistake in `PROVIDES`:
```diff
 PROVIDES="
 	retroarch$secondaryArchSuffix = $portVersion
-	app:retroarch$secondaryArchSuffix = $portVersion
+	app:retroarch = $portVersion
-	cmd:retroarch$secondaryArchSuffix = $portVersion
-	cmd:retroarch_cg2glsl$secondaryArchSuffix = $portVersion
+	app:retroarch_cg2glsl = $portVersion
 	"
```

Reply:
Is this a new requirement? We never had it before afaik

Reply:
Ok, give me a minute and I'll push a new commit